stories/index.stories.js

```diff
@@ -123,6 +123,37 @@ export const S2GetMapProcessing = () => {
   return img;
 };
 
+export const S1GetMapProcessingFromLayer = () => {
+  if (!process.env.STORYBOOK_AUTH_TOKEN) {
+    return '<div>Please set auth token for Processing API (STORYBOOK_AUTH_TOKEN env var)</div>';
+  }
+  setAuthToken(process.env.STORYBOOK_AUTH_TOKEN);
+
+  const img = document.createElement('img');
+  img.width = '512';
+  img.height = '512';
+
+  // getMap is async:
+  const perform = async () => {
+    const layer = new S1GRDIWAWSLayer(instanceId, 'S1GRDIWDV');
```

Comment:
Can't make this example work even after creating the layer with this name and trying all possible polarizations and scripts.

Reply:
Yes, this is a problem of V2/V3 scripts and dataproducts. The dataProduct you have selected is not compatible with Processing API (yet - I assume), so you need to enter an evalscript. Note that it must be V3, for example:

```
//VERSION=3
function setup() {
  return {
    input: ["VV"],
    output: { bands: 3 }
  };
}

function evaluatePixel(sample) {
  return [
    2 * sample.VV,
    2 * sample.VV,
    2 * sample.VV,
  ];
}
```

Note that polarization needs to be `DV` (`VV+VH`) for this storybook to work.

Reply:
Thanks for the answer. Your evalscript works. In was indeed the problem with the version.